site/gatsby-site/server/index.ts

```diff
@@ -0,0 +1,15 @@
+import { startStandaloneServer } from '@apollo/server/standalone';
```

Comment:
This file is used (for now) to start a local API server for development purposes only.

`npm run start:api`